### PR TITLE
Issue 380

### PR DESCRIPTION
**Pull request by tfausak**

### CODE REVIEW

**Comment by tfausak** [comment]
These were posted before I published issue 379, but weren't included in that issue. I didn't notice them until too late:

- https://discourse.haskell.org/t/iog-ghc-update-15-iog-engineering/7191
- https://discourse.haskell.org/t/haskell-foundation-july-2023-update/7192

File: data/newsletter/issue-380.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+- [Reducing Haskell parallel build times using semaphores](https://well-typed.com/blog/2023/08/reducing-haskell-parallel-build-times/) by Well-Typed
+> This blog post explains a new feature for GHC & Cabal, which allows multiple simultaneously running GHC processes to coordinate their use of compute resources.
+
+- [Speeding up tree-sitter-haskell 50x](https://owen.cafe/posts/tree-sitter-haskell-perf/) by Owen Shepherd
+> This article is about profiling and picking the low-hanging fruit of tree-sitter-haskell (warning: C(++)).
+
+## Jobs
+
+undefined
```
Comment (tfausak) [inline data/newsletter/issue-380.markdown:30]: 😜 

I was talking with Evgeny about running an ad for SimpleX Chat. I was going to add it this morning, but this PR had already been merged! I added it anyway: b8de2d8cee24d23b50118a3df72881f187cd8535. Still before the emails go out, so all is good.

Comment (alexeyzab) [reply data/newsletter/issue-380.markdown:30]: Oh, my bad. Should I hold off merging the PR until this section is filled out next time?

Comment (tfausak) [reply data/newsletter/issue-380.markdown:30]: Nah, I'll be sure to let you know when an ad is possible.

**Pull request merged**